update.js

```diff
@@ -1,18 +1,16 @@
-
 const NavBar: FC = () => {
     const [pageSwtichButtons, setPageSwitchButtons] = useState(
       InitialPageSwitchButtons
     );
-    const [currentButton, setCurrentButton] = useState(
-      InitialPageSwitchButtons[0]
-    );
+dfsdfsd
   
     return (
         <div className="d-flex flex-row align-items-center navbar-desktop-right-content">
```

Comment:
<div><div>It seems that your code expects the byte array to represent a 32bit integer in big-endian byte order. We assume that for some reason if you run your code under linux, the byte order is little-endian. We would need to know where the byte array comes from to guess a reason for this.<br/><br/></div><div><img align="left" src="https://picsum.photos/200/200" width="100" height="100" title="My Pic"><b>Name</b> <sub>Just Now</sub><br/>Good Catch! I will go ahead and change this now. For now, we are not worried about endian, We will change it after. Thanks for the suggestion. Do you have any recommendations for os specific task that we may need to change further? I've been thinking about shifting some of our codebases to cloud services Also Is there any alternative to google Map API for showing maps and nearest places?</div></div>